http.go

```diff
@@ -1290,13 +1291,19 @@ func (resp *Response) ReadLimitBody(r *bufio.Reader, maxBodySize int) error {
 
 	if !resp.mustSkipBody() {
 		err = resp.ReadBody(r, maxBodySize)
+		if errors.Is(err, syscall.ECONNRESET) {
```

Comment:
Can be embedded into next if err != nil block

Reply:
Did change this

Reply:
@ArminBTVS please check latest comments